### PR TITLE
Code Fix for ISSUE 2092 & 2064

### PR DESCRIPTION
**Pull request by marut-launch**
Fix for below issues
https://github.com/sequenceiq/cloudbreak/issues/2092
https://github.com/sequenceiq/cloudbreak/issues/2064

Fix summary

Added new form under "Manage Security group" to add existing groups along with new security group.
Added 2 new columns to Securitygroup table, for adding cloudplatform and storing cloud existing security group id.
Added ability to provide custom image id for building cluster.

### CODE REVIEW

**Comment by doktoric** [comment]
@marut-launch You should also update this class https://github.com/sequenceiq/cloudbreak/blob/master/core/src/main/java/com/sequenceiq/cloudbreak/service/securitygroup/DefaultSecurityGroupCreator.java

File: shell/src/main/java/com/sequenceiq/cloudbreak/shell/commands/common/SecurityGroupCommands.java [hunk 1]
```diff
@@ -36,44 +36,53 @@ public boolean createAvailable() {
     public String create(
             @CliOption(key = "name", mandatory = true, help = "Name of the security group") String name,
             @CliOption(key = "description", help = "Description of the security group") String description,
-            @CliOption(key = "rules", mandatory = true,
+            @CliOption(key = "existingGpId", help = "Existing security group id.") String existingSecurityGroupId,
```
Comment (doktoric) [inline shell/src/main/java/com/sequenceiq/cloudbreak/shell/commands/common/SecurityGroupCommands.java]: this class is not platform specific. Please check how we implemented like the network commands.

**Comment by doktoric** [comment]
@marut-launch You should also create the securitygroup for the mock integration tests. Here is the security group which is not exist for the tests 
https://github.com/sequenceiq/cloudbreak/blob/master/integration-test/src/main/resources/application.yml#L99

File: shell/src/main/java/com/sequenceiq/cloudbreak/shell/commands/common/SecurityGroupCommands.java (hunk 1 above)
Comment (marut-launch) [inline shell/src/main/java/com/sequenceiq/cloudbreak/shell/commands/common/SecurityGroupCommands.java]: Updated securitygroup commands specific to platform.

**Comment by marut-launch** [comment]
Updated code according to review comments, please review.

**Pull request merged**